new/prusti-specs/src/specifications/untyped.rs

```diff
@@ -86,6 +83,39 @@ impl AssignExpressionId<Expression> for common::Expression<(), syn::Expr> {
     }
 }
 
+impl AssignExpressionId<ForAllVars<ExpressionId, Arg>> for common::ForAllVars<(), Arg> {
+    fn assign_id(
+        self,
+        spec_id: SpecificationId,
+        id_generator: &mut ExpressionIdGenerator,
+    ) -> ForAllVars<ExpressionId, Arg> {
+        ForAllVars {
+            id: id_generator.generate(),
+            vars: self.vars
+        }
+    }
+}
+
+impl AssignExpressionId<TriggerSet> for common::TriggerSet<(), syn::Expr> {
+    fn assign_id(
+        self,
+        spec_id: SpecificationId,
+        id_generator: &mut ExpressionIdGenerator,
+    ) -> TriggerSet {
+        TriggerSet {
+            0: self.0
```

Comment:
```suggestion
  TriggerSet(self.0
```

Reply:
i will resolve this once this whole part of the code is refactored, as it is probably incorrect anyway

Reply:
Fine with me.